webviz_subsurface/containers/_view3d.py

```diff
@@ -0,0 +1,182 @@
+        return self.config['wells']
+
+    @property
+    def center_x(self):
+        return self.config['center_x']
+
+    @property
+    def center_y(self):
+        return self.config['center_y']
+
+    @property
+    def extent(self):
+        return self.config['extent']
+
+    @property
+    def inc(self):
+        return self.config['increment']
+
+    @property
+    def path(self):
+        return self.config['path']
+
+    @property
+    def style_modal(self):
+        return {
+            'position': 'fixed',
+            'top': '5%',
+            'left': '20%',
+            'z-index': '1000',
+            'width': '20%',
+            'height': '20%',
+        }
+
+    def selector(self, id, values, multi=False):
+        return dcc.Dropdown(
+            # inputStyle={'display':'table-'},
+            id=id,
+            options=[{'label': s, 'value': s}
+                     for s in values],
+            clearable=False,
+            value=values[0],
+            multi=multi
+        )
+
+    @property
+    def selectors(self):
+        layout = []
+        layout.append(html.H3('Surfaces'))
+        for surf_type, surfs in self.surfaces.items():
+            layout.append(html.P(surf_type))
+            layout.append(self.selector(
+                f'{surf_type}--{self.selectors_id}', surfs, multi=False))
+        layout.append(html.H3('Wells'))
+        layout.append(self.selector(
+            f'wells--{self.selectors_id}', self.wells, multi=True))
+        return layout
+
+    @property
+    def layout(self):
+        return html.Div(children=[
+            html.Div(
+                id=self.modal_id,
+                children=[
+                    html.Div(
+                        id=self.selectors_id,
+                        children=self.selectors,
+                        style=self.style_modal)
+                ]),
+            html.Div(children=[wsc.View3D(
+                id=self.vizvaz_id,
+                center_x=self.center_x,
+                center_y=self.center_y)])
+        ])
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output(self.vizvaz_id, 'surface'),
+            [Input(f'depth--{self.selectors_id}', 'value')])
+        def update_surface(s):
+            surf_data = get_surface_data(self.path, s, 'depth', self.center_x,
+                                         self.center_y, self.extent, self.inc)
+            return surf_data
+
+        @app.callback(
+            Output(self.vizvaz_id, 'wells'),
+            [Input(f'wells--{self.selectors_id}', 'value')])
+        def update_wells(wells):
+            well_data = get_well_path_data(self.path, wells)
+            return well_data
+
+    def add_webvizstore(self):
+        funcs = []
+        funcs.append((get_path,
+                      [{'config': self.config_path}]))
+        for surf_type, surfs in self.surfaces.items():
+            for surf in surfs:
+                path = os.path.join(self.path, 'maps',
```

Comment:
Is `self.path` defined (can't find it by searching in the PR).

If it is an object from `pathlib` you can use [the more readable `/` operator from `pathlib` to join paths](https://docs.python.org/3/library/pathlib.html#operators), and remove the need for `import os` as well.